### DISCUSSION TITLE
Option to toggle component sizes on demo pages

### DISCUSSION THREAD

**Discussion by FlyersPh9** [iTwin/iTwinUI#102] [Ideas]
As we are starting to add more sizes to most of our components it would be nice to have a row of links underneath the demo page, similar to this:
![Screen Shot 2021-05-20 at 12 14 34 PM](https://user-images.githubusercontent.com/849817/119013415-ce508000-b94b-11eb-8212-33069ff6c928.png)

This would cut down on the amount of components per demo page.  In some cases like text field I could see this being very nice.

Two issues i foresee with this:
1. **Testing:**  we would need tests to run 3x to capture all the different sizes separately.
2. **Nested objects:**  the text field with a borderless button right aligned within it.  if the text field changes to `.iui-large` the button also needs that class appended, not just the text field.

**Comment by maxime4000** [comment]
## I made this month ago, would be nice to see it come to life. 
![FlextableTrueInteractive](https://user-images.githubusercontent.com/4070713/119020349-2303f300-b96c-11eb-9747-52a20c228925.gif)
